app/components/elements/delete_button_component.rb

```diff
@@ -6,5 +6,9 @@ class DeleteButtonComponent < IconButtonComponent
     def initialize(label: 'Clear', **args)
       super(icon: :delete, label:, **args)
     end
+
+    def classes
+      merge_classes(super, @classes, %w[px-0])
```

Comment:
@justinlittman Including `@classes` here is not sufficient to pull in the classes merged via the parent class, `IconButtonComponent`, and when I tested, `@classes` was an empty array. I added `super` here to make sure the `border` and `border-0` classes are injected properly via the parent. Not sure if this is a pattern we want to be using or not.

Reply:
I'd be inclined to handle it in the initializer:
```
def initialize(label: 'Clear', classes: [], **args)
      super(icon: :delete, label:, classes: Array(classes) + ['px-0'], **args)
    end
```

How does that seem?